tests/pytest.ini

```diff
@@ -7,3 +7,5 @@ markers:
     disable_loganalyzer: make to disable automatic loganalyzer
     broadcom: test specific to Broadcom platform
     sanity_check: override the default sanity check settings
+    topology_any: test that can run on any topology (t0, t1, ptf, etc)
```

Comment:
This is a question for @neethajohn but I thought the idea was to define one topology marker and pass arguments to it like `topology(t0)`, `topology(t1)`, etc.

Reply:
Yes. topology(name) is what we thought of having. name can be 't0', 't1', 'ptf', 'any'

Reply:
I am waiting until we have figured out the plumbing of using topology(name) format. Do we need to also define an filter argument for pytest to be able to pick up this marker? Or do we feel that we should just go to this format now?

Reply:
Changed to our intended format.